app/views/invites/new.html.erb

```diff
@@ -22,19 +22,23 @@
               <%= form_for([@proposal,@invite]) do |f| %>
                 <div class="mb-3">
                   <label>First Name</label>
-                  <%= f.text_field :firstname, autofocus: true, class:'form-control' %>
+                  <%= f.text_field :firstname, autofocus: true, class:'form-control', required: true %>
                 </div>
                 <div class="mb-3">
                   <label>Last Name</label>
-                  <%= f.text_field :lastname, class:'form-control' %>
+                  <%= f.text_field :lastname, class:'form-control', required: true %>
                 </div>
                 <div class="mb-3">
                   <label>Email</label>
-                  <%= f.email_field :email, class:"form-control form-control-lg", placeholder:"Enter your email", autocomplete: "email" %>
+                  <%= f.email_field :email, class:"form-control form-control-lg", placeholder:"Enter your email", autocomplete: "email", required: true %>
                 </div>
-                <div>
+                <div class="mb-3">
                   <label>Invite as</label>
-                  <%= f.select(:invited_as, options_for_select([['Co Organizer', 'Co Organizer'], ['Participant', 'Participant']]), {}, {class: "form-control"}) %>
+                  <%= f.select(:invited_as, options_for_select([['Co Organizer', 'Co Organizer'], ['Participant', 'Participant']]), {required: true}, {class: "form-control"}) %>
+                </div>
+                <div class="mb-3">
+                  <%= f.label :deadline %>
+                  <%= f.date_field :deadline_date,  class:"form-control", required: true %>
                 </div>
                 <div class="text-center mt-3">
                   <%= f.submit "Send", class: 'btn btn-lg btn-primary' %>
```

Comment:
"Co Organizer" should be replaced by "Co-Organizer". The database should save it as "Organizer", for compatibility with other BIRS systems. Similarly with "Lead Organizer", should be saved as "Contact Organizer".